Update quickstart with cargo stylus check timing and Docker requirements (#1635)
OffchainLabs/arbitrum-docs: arbitrum-docs/stylus/stylus-quickstart.md

Program succeeded Stylus onchain activation checks with Stylus version: 1
```

> Note that running `cargo stylus check` may take a few minutes, especially on a fresh build. If you're running the check or deploy commands, a message at the start will inform you that this process might take some time. For faster checks, you can use the `--no-verify` option. Additionally, please ensure that Docker is up and running as it's required for these commands to execute properly.

Reviewer: ```suggestion
Note that running `cargo stylus check` may take a few minutes, especially on a fresh build. If you're running the check or deploy commands, a message at the start will inform you that this process might take some time. For faster checks, you can use the `--no-verify` option. Also, please ensure that Docker is up and running, as these commands are required to be executed properly.
```
